amazon-massive-pl.py

```diff
@@ -310,22 +324,19 @@ def get_categories_as_string(categories: List,
 
 	if len(categories) > 10 and main_category: 
 		x = 0
+		random_categories = random.choices(list(categories), k=5)
```

Comment:
This change should not happen as it does crash the function. The `random.choices()` shall remain where it was previously.